brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-634 - Improvement : Taxonomy List - Removed highlighted help text from some option.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/b57bda09-fae3-40bf-aa19-8155e8923648?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- https://share.getcloudapp.com/RBunQbAw

### Types of changes




- Improvement.

### How has this been tested?

- Drag and drop the taxonomy list.
- Check Show Empty Taxonomy, Show Posts Count & Show Hierarchy option
- Highlighted help text is removed as it is not needed anymore.

### Checklist:
